src/VisualStudio/Core/Test/Snippets/SnippetTestState.vb

```diff
@@ -148,6 +138,58 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Snippets
             MyBase.SendTypeChars(typeChars, AddressOf handler.ExecuteCommand)
         End Sub
 
+        Public Async Function AssertNoCompletionSession() As Task
+            Await WaitForAsynchronousOperationsAsync()
+            Dim session = GetExportedValue(Of IAsyncCompletionBroker)().GetSession(TextView)
+            If session Is Nothing Then
+                Return
+            End If
+
+            If session.IsDismissed Then
+                Return
+            End If
+
+            Dim completionItems = session.GetComputedItems(CancellationToken.None)
+            ' During the computation we can explicitly dismiss the session or we can return no items.
+            ' Each of these conditions mean that there is no active completion.
+            Assert.True(session.IsDismissed OrElse completionItems.Items.Count() = 0, "AssertNoCompletionSession")
```

Comment:
```suggestion
            Assert.True(session.IsDismissed OrElse completionItems.Items.Count() = 0, NameOf(AssertNoCompletionSession))
```

Reply:
Error BC36946 The implicit return variable of an Iterator or Async method cannot be accessed. 


---
In reply to: [322474684](https://github.com/dotnet/roslyn/pull/38569#discussion_r322474684) [](ancestors = 322474684)